src/interfaces/filing-interfaces/filing-interface.ts

```diff
@@ -87,23 +88,19 @@ export interface RegistrationFilingIF {
     priority?: boolean
   },
   registration: {
-    startDate: string
-    // NB: nameRequest must match schema
-    nameRequest: {
-      legalType: CorpTypeCd
-      nrNumber?: string // only set when there is an NR
-      legalName?: string // only set when there is an NR
-    },
-    businessAddress: BusinessAddressIF,
+    business: {
+      identifier: string
+    }
+    businessAddress: BusinessAddressIF
     contactPoint: {
       email: string
-      phone: string
       extension?: number
-    },
-    parties: OrgPersonIF[],
-    business: {
-      identifier: string
+      phone: string
     }
+    naics?: RegistrationNaicsIF
```

Comment:
I think @vysakh-menon-aot mentioned this, `naics` belongs to the `business`

Wayyyyyy at the bottom:
https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/business.json

Reply:
Yes, oops, fixed in new commit 😁